erpnext/projects/doctype/timesheet/timesheet.js

```diff
@@ -294,4 +298,17 @@ var calculate_time_and_amount = function(frm) {
 	frm.set_value("total_hours", total_working_hr);
 	frm.set_value("total_billable_amount", total_billable_amount);
 	frm.set_value("total_costing_amount", total_costing_amount);
-}
\ No newline at end of file
+};
+
+// set employee (and company) to the one that's currently logged in
+const set_employee_and_company = function(frm) {
+	const options = { user_id: frappe.session.user };
+	const fields = ['name', 'company'];
+	frappe.db.get_value('Employee', options, fields).then(({ message }) => {
+		if (message) {
+			// there is an employee with the currently logged in user_id
+			frm.fields_dict['employee'].set_value(message.name);
```

Comment:
Let's use `frm.set_value` for consistency instead?
https://github.com/frappe/erpnext/pull/16743/files#diff-bb8ecad0277db241f5eef7705751b0ceR300

Reply:
Okay, but only if you promise to get this merged within a week :stuck_out_tongue_winking_eye: